client/me/billing-history/transactions-table.jsx

```diff
@@ -105,19 +105,20 @@ class TransactionsTable extends React.Component {
 	}
 
 	serviceName = transaction => {
-		var item, name;
-
 		if ( ! transaction.items ) {
-			name = this.serviceNameDescription( transaction );
-		} else if ( transaction.items.length === 1 ) {
-			item = transaction.items[ 0 ];
-			item.plan = capitalPDangit( titleCase( item.variation ) );
-			name = this.serviceNameDescription( item );
-		} else {
-			name = <strong>{ this.props.translate( 'Multiple items' ) }</strong>;
+			return this.serviceNameDescription( transaction );
 		}
 
-		return name;
+		const groupedTransactionItems = groupDomainProducts( transaction.items, this.props.translate );
```

Comment:
Can `groupedTransactionItems` ever be an empty array?

Reply:
Yes, if the receipt has no items - which I'd say it'd be quite a corner case.